socketio/examples/callback.rs

```diff
@@ -13,9 +13,10 @@ fn main() {
         match payload {
             Payload::String(str) => println!("Received string: {}", str),
             Payload::Binary(bin_data) => println!("Received bytes: {:#?}", bin_data),
+            Payload::Number(num) => println!("Received number: {}", num),
```

Comment:
What value does this type add? Isn't it already covered by binary?

Reply:
currently rust_socketio can not emit `number` value, it will encode as string